esmvaltool/cmorizers/obs/cmorize_obs_woa.py

```diff
@@ -75,10 +89,12 @@ def cmorization(in_dir, out_dir, cfg, _):
     # run the cmorization
     for var, vals in cfg['variables'].items():
         for yr in cfg['custom']['years']:
-            file_suffix = str(yr)[-2:] + '_' + str(yr + 1)[-2:] + '.nc'
-            inpfile = os.path.join(in_dir, vals['file'] + file_suffix)
+            #file_suffix = str(yr)[-2:] + '_' + str(yr + 1)[-2:] + '.nc'
+            #inpfile = os.path.join(in_dir, vals['file'] + file_suffix)
```

Comment:
if these lines are not needed as in-line comments please remove them, if they are, please add a human-readable explanation and use convention `#[space][text]` for style :+1:

Reply:
I deleted the lines.

Reply:
cheers Lisa!